commands/core/drupal/update.inc

```diff
@@ -196,6 +205,15 @@ function drush_update_batch() {
     }
   }
 
+  // Apply post update hooks.
+  $post_updates = \Drupal::service('update.post_update_registry')->getPendingUpdateFunctions();
+  if ($post_updates) {
+    $operations[] = ['drupal_flush_all_caches', []];
```

Comment:
Hmmm.

1. I'm not sure why this needs to be added. I fear that we are going to clear caches twice. Does Drupal UI do that?
1. We always clear caches in a separate process now which can be skipped via an option. You can replace drupal_flush_all_caches() with drush_drupal_cache_clear_all().